<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceHistoryCompare</class>
 <widget class="QWidget" name="FaceHistoryCompare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:0px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(235, 235, 235, 255), stop:1 rgba(170, 170, 170, 255));</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>2</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../Resouce.qrc">:/images/身份验证.png</pixmap>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>2</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;
font: 11pt &quot;黑体&quot;;</string>
         </property>
         <property name="text">
          <string>身份验证</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">border:1px solid rgb(180,180,180);
border-radius:0px;
background-color:rgb(236,236,236)</string>
        </property>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>648</width>
           <height>655</height>
          </rect>
         </property>
         <widget class="QWidget" name="widget_6" native="true">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>80</y>
            <width>271</width>
            <height>381</height>
           </rect>
          </property>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>20</y>
             <width>91</width>
             <height>20</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;
font: 11pt &quot;黑体&quot;;</string>
           </property>
           <property name="text">
            <string>请添加照片</string>
           </property>
          </widget>
          <widget class="QLabel" name="m_laelImage">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>60</y>
             <width>210</width>
             <height>240</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="m_btnSelectImage">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>310</y>
             <width>147</width>
             <height>58</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/images/选择照片-01.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="widget_7" native="true">
          <property name="geometry">
           <rect>
            <x>330</x>
            <y>80</y>
            <width>271</width>
            <height>381</height>
           </rect>
          </property>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>20</y>
             <width>151</width>
             <height>20</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;
font: 11pt &quot;黑体&quot;;</string>
           </property>
           <property name="text">
            <string>历史人脸库条件筛选</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="m_editArea">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>60</y>
             <width>181</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>140</y>
             <width>21</width>
             <height>16</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;
font: 10pt &quot;黑体&quot;;</string>
           </property>
           <property name="text">
            <string>至</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_24">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>60</y>
             <width>31</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;
font: 10pt &quot;黑体&quot;;</string>
           </property>
           <property name="text">
            <string>区域</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_25">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>100</y>
             <width>31</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;
font: 10pt &quot;黑体&quot;;</string>
           </property>
           <property name="text">
            <string>时间</string>
           </property>
          </widget>
          <widget class="QDateTimeEdit" name="m_dtBeginTime">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>100</y>
             <width>181</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QDateTimeEdit" name="m_dtEndTime">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>170</y>
             <width>181</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>240</y>
             <width>41</width>
             <height>16</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;
font: 10pt &quot;黑体&quot;;</string>
           </property>
           <property name="text">
            <string>已选择</string>
           </property>
          </widget>
          <widget class="QWidget" name="widget_8" native="true">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>260</y>
             <width>71</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QWidget" name="widget_9" native="true">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>300</y>
             <width>141</width>
             <height>51</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QPushButton" name="m_btnBeginCompare">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>520</y>
            <width>205</width>
            <height>76</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/images/btnBeginCompare-01.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_22">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>30</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none;
font: 14pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>对比阈值设置</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_23">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>22</y>
            <width>54</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none;
font: 18pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="m_spinValue1">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>30</y>
            <width>40</width>
            <height>30</height>
           </rect>
          </property>
         </widget>
         <widget class="QSpinBox" name="m_spinValue2">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>30</y>
            <width>40</width>
            <height>30</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="widget_5" native="true">
         <property name="geometry">
          <rect>
           <x>647</x>
           <y>0</y>
           <width>433</width>
           <height>655</height>
          </rect>
         </property>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>30</y>
            <width>101</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none;
font: 16pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>比对结果</string>
          </property>
         </widget>
         <widget class="QPushButton" name="m_btnRunTrack">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>580</y>
            <width>220</width>
            <height>60</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/images/生成行动轨迹图-01.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QTableWidget" name="m_tableCompareResult">
          <property name="geometry">
           <rect>
            <x>1</x>
            <y>81</y>
            <width>431</width>
            <height>481</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resouce.qrc"/>
 </resources>
 <connections/>
</ui>
